JavaScript/JavaScriptSDK/Initialization.ts

```diff
@@ -169,6 +169,10 @@ module Microsoft.ApplicationInsights {
             config.disableCorrelationHeaders = (config.disableCorrelationHeaders !== undefined && config.disableCorrelationHeaders !== null) ?
                 Util.stringToBoolOrDefault(config.disableCorrelationHeaders) :
                 true;
+
+            config.disableFlushOnBeforeUnload = (config.disableFlushOnBeforeUnload !== undefined && config.disableFlushOnBeforeUnload !== null) ?
+                Util.stringToBoolOrDefault(config.disableFlushOnBeforeUnload) :
```

Comment:
Seems like Util.stringToBoolOrDefault could handle null and undefined cases - would have saved us some typing. Or if you are trying to check for null and undefined why not just check if (config.disableFlushOnBeforeUnload){...}